pkg/Microsoft.Private.PackageBaseline/baseline.packages.props

```diff
@@ -423,6 +423,9 @@
     <BaseLinePackage Include="runtime.native.System.Security.Cryptography">
       <Version>4.0.0</Version>
     </BaseLinePackage>
+    <BaseLinePackage Include="runtime.native.System.Security.Cryptography.Apple">
+      <Version>4.0.1</Version>
```

Comment:
Mainly that I had copied the already-bumped runtime.native.System.Security.Cryptography package when building the Apple one (which is already on myget) and it hadn't seemed worthwhile to lower it (and it seemed nice and symmetric to have them match).